drivers/hmis/spec/factories/hmis/form/assessment_details.rb

```diff
@@ -7,9 +7,8 @@
 FactoryBot.define do
   factory :hmis_form_custom_form, class: 'Hmis::Form::CustomForm' do
```

Comment:
It doesn't look like this was used anywhere and it seemed like all the fields were pasted in from somewhere else (maybe FormDefinition?). I went ahead and reworked it so that it works as expected, but let me know if I shouldn't have done that.

Reply:
Ah thanks for fixing it. It actually shouldn't be linked to a `hmis_hud_assessment`, it should use the "hmis_custom_assessment" factory. "HUD asssessments" are Coordinated Entry assessments, which we're not using at all yet. The way you set it up in the spec, passing a CustomAssessment as the owner, looks correct.

FYI the reason this factory wasn't used is because in most other places we use the "hmis_custom_assessment_with_defaults" to create assessments, which automatically creates the CustomForm alongside it.